source/global_variables.cpp

```diff
@@ -3,4 +3,5 @@
 namespace g {
     bool network_init = false;
     TextRenderer* font_renderer = NULL;
+    unsigned int bar, dark_bar;
```

Comment:
i think a textures namespace would be better,
and doesn't the texture manager of stardust handle loading the same texture?

Reply:
Not currently - but will in the future. I do have to find a way to avoid variants loading - i.e. "ms0:/path/to/image" "./image.png" "image.png" etc. all refer to the same file.